tests/conftest.py

```diff
@@ -95,3 +95,24 @@ class Starter(ProcessStarter):
 
     yield
     xprocess.getinfo("memcached").terminate()
+
```

Comment:
While I really like tests, I don't think that they should be included in the official test suite as we would have to maintain them here.